src/blockchain/src/block.rs

```diff
@@ -75,7 +79,16 @@ impl Block {
     }
 }
 
-// struct Transaction define the details of a transaction in a block
+/// ## Transactions ##
+/// A Transaction consists of
+/// - `trans_type` Pretty obvious. Used to help decode the rest of the transaction
```

Comment:
These should go over the fields

Reply:
I am going to take this a step further. Ironically, we've outsourced our code formatting to a group of people we've never met and had no say in the process, but the documentation is totally freestyle. 

I am literally going to rip off the format in the rust core libs

Reply:
your comments made no sense... it's already specd on how it should be written

https://doc.rust-lang.org/rust-by-example/meta/doc.html

https://doc.rust-lang.org/rustdoc/index.html

Reply:
I don't see the part about how structs should be documented. But, it's fine. I have implemented as you requested.